src/controllers/PostController.ts

```diff
@@ -1,32 +1,33 @@
-import {JsonController, Get, Post as HttpPost, Param, Delete, Body} from "routing-controllers";
+import {JsonController, Get, Post, Param, Delete, Body} from "routing-controllers";
 import {Service} from "typedi";
 import {PostRepository} from "../repository/PostRepository";
-import {Post} from "../model/Post";
+import {Post as PostModel} from "../model/Post";
```

Comment:
Best if we won't use Post in the example at all. Use some different domain. For example `Question` instead of `Post`.